Code/Framework/AzFramework/Platform/Windows/AzFramework/Device/DeviceAttributesCommon_Win.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformIncl.h>
+#include <AzCore/std/string/conversions.h>
+#include <AzFramework/Device/DeviceAttributes.h>
+
+namespace AzFramework
+{
+    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    {
+        HKEY hKey = nullptr;
+        LSTATUS returnCode = ::RegOpenKeyEx(HKEY_LOCAL_MACHINE, TEXT("HARDWARE\\DESCRIPTION\\System\\BIOS"), 0, KEY_READ, &hKey);

```

Comment:
Protip: You can use raw string literals for Windows registry keys to avoid the need for backslash escaping.
```suggestion
        LSTATUS returnCode = ::RegOpenKeyEx(HKEY_LOCAL_MACHINE, TEXT(R"(HARDWARE\DESCRIPTION\System\BIOS)"), 0, KEY_READ, &hKey);

```